package/MDAnalysis/coordinates/DCD.py

```diff
@@ -191,7 +191,7 @@ def _read_next_timestep(self, ts=None):
         if ts is None:
             # use a copy to avoid that ts always points to the same reference
             # removing this breaks lammps reader
-            ts = self.ts.copy()
+            ts = self.ts.copy() # why is this copy required ??
```

Comment:
@orbeckst @MDAnalysis/coredevs does anyone know specifically why this copy is required? None of the other readers use this pattern and it is ~30% overhead but removing it breaks a bunch of tests. I will keep working on it but if anyone has a quick answer would be appreciated.

Reply:
I will keep working on it in the meantime.

Reply:
I am going to move discussion to an issue.

Reply:
Moved to #3889